File: 240621/문자열 길이의 합/sum-length-of-string.py
N = int(input())
input_strs = [input() for _ in range(N)]

total_lengths, a_cnts = 0, 0
for input_str in input_strs:
    total_lengths += len(input_str)
    if input_str.startswith('a'):
        a_cnts += 1

print(total_lengths, a_cnts)

File: 251129/문자열 길이의 합/sum-length-of-string.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int N;
    cin >> N;
    cin.ignore();   

    vector<string> input_strs(N);
    for (int i = 0; i < N; i++) 
    {
        getline(cin, input_strs[i]);
    }

    int total_lengths = 0;
    int a_cnts = 0;

    for (const string &s : input_strs) 
    {
        total_lengths += s.length();
        if (!s.empty() && s[0] == 'a') 
        {
            a_cnts++;
        }
    }

    cout << total_lengths << " " << a_cnts;
    return 0;
}
